README.md

```diff
@@ -159,16 +165,16 @@ r.insert(ast::forall(|[p, q, r]| {
     )
     .when(add, vec![p.into(), q.into(), r.into()])
 }));
-```
 
-We can now ask the solver to prove statements within this universe, e.g. that "there exists an X
-such that X + 2 = 3". This statement is indeed true for X = 1, and indeed, the solver will provide
-this answer:
+// We can now ask the solver to prove statements within this universe, e.g. that "there exists an X
```

Comment:
The easiest way to make `cargo test` pass was to merge these blocks. We are traiding flexibility of writing for correctness, though.